src/main/java/com/github/davidmoten/rx2/internal/flowable/FlowableCollectWhile.java

```diff
@@ -46,7 +47,7 @@ protected void subscribeActual(Subscriber<? super R> child) {
 
     @SuppressWarnings("serial")
     private static final class CollectWhileSubscriber<T, R> extends AtomicInteger
```

Comment:
This is why we need a rebase from master. `master` uses FlowableSubscriber already (https://github.com/davidmoten/rxjava2-extras/blob/master/src/main/java/com/github/davidmoten/rx2/internal/flowable/FlowableCollectWhile.java).